README.md

```diff
@@ -230,6 +242,21 @@ Will safely cleanup any data leftover in the `call` object.
 
 This example can be found in examples/client/client-insecure.lisp.
 
+### Server
+
+A minimal gRPC service with protobuf integration looks like:
+
+```
+; load the protobuf-generated code here
+
+(defmethod cl-protobufs.testing-rpc::say-hello ((request cl-protobufs.testing:hello-request) call)
```

Comment:
I opted to use the fully-qualified names since someone who is testing this in the REPL would have to use those (as opposed to local-nicknames as part of a `defsystem`. Let me know your preference and I'll go double check that `say-hello` is exported.

Reply:
Aha, you've been looking at `examples/client/client-insecure.lisp` which uses `examples/client/helloworld.proto` which defines `package lisp.grpc.integration_testing`. However, I've been looking a few lines up in the README, which defines `package testing`. This is why we disagree about the namespace.

I do think there is value in providing a minimum working example so if there are no objections I'll add that.